components/formstack/sources/common/common.mjs

```diff
@@ -0,0 +1,52 @@
+import formstack from "../../formstack.app.mjs";
```

Comment:
One convention I've used so far in terms of naming common files is that if you are in the `common` folder you should start with the `base.mjs` file name. So I would recommed rename `common/common.mjs` by `common/base.mjs`